soh/soh/Enhancements/cosmetics/CosmeticsEditor.h

```diff
@@ -8,6 +8,28 @@
         ResourceMgr_UnpatchGfxByName(path, name); \
     }
 
+// Not to be confused with tabs, groups are 1:1 with the boxes shown in the UI, grouping them allows us to reset/randomize
+// every item in a group at once. If you are looking for tabs they are rendered manually in ImGui in `DrawCosmeticsEditor`
+typedef enum {
+    GROUP_LINK,
```

Comment:
With these moved to a header, they should be prefixed with `COSMETICS_` for clarity/avoid possible collisions when imported.
(also good to add a `_MAX` at the end)